pgo-scheduler/pgo-scheduler.go

```diff
@@ -53,7 +53,7 @@ var namespace string
 var pgoNamespace string
 var timeout time.Duration
 var seconds int
-var kubeClient kubernetes.Interface
+var clientset kubeapi.Interface
```

Comment:
This is one change to the client in scheduler. It is initialized in the next hunk in this same file.